server_admin/topics/clients/oidc/proc-using-a-service-account.adoc

```diff
@@ -0,0 +1,65 @@
+[id="proc-using-a-service-account_{context}"]
+[[_service_accounts]]
+
+==== Using a Service Account
+
+Each OIDC client has a built-in _service account_ which allows it to obtain an access token.
+
+.Prerequisites
+* You have selected your client in {project_name}.
```

Comment:
You could think about making these steps 1 and 2 of the procedure.  I generally use prerequisites to help break up the procedure.  The three of us should probably talk about this.